app/controllers/mixins/actions/provider_actions/mass_transform.rb

```diff
@@ -14,7 +14,7 @@ def vm_transform_mass
               :target_kls => Provider.name
             )
           else
-            provider = ExtManagementSystem.find_by!(:type => 'ManageIQ::Providers::Redhat::InfraManager')
+            provider = ExtManagementSystem.find_by!(:type => 'ManageIQ::Providers::Ovirt::InfraManager')
```

Comment:
Technically this would have to be `provider = ExtManagementSystem.find_by!(:type => %w[ManageIQ::Providers::Ovirt::InfraManager ManageIQ::Providers::Redhat::InfraManager])`.   This code is already bad and I think not used anymore without the v2v plugin so don't think we want to spend the time improving this, more likely we'll delete it shortly